draft-ietf-oauth-identity-chaining.md

```diff
@@ -265,6 +265,8 @@ Cache-Control: no-cache, no-store
 
 ## Claims transcription
 
+Claims transcription is motivated by the need to propagate user and client identities, and potentially other information, across different trust domains when a request traverses multiple protected resources. This ensures that each resource knows on whose behalf the request is being made, what authorization is granted, and potentially which other resources were previously involved.
```

Comment:
propagate user and client identities, authorisation context and potentially other information